docs/man/ctags.1.rst

```diff
@@ -1721,26 +1717,41 @@ The meaning of major fields is as follows (one-letter flag/long-name flag):
 	callable like function as ``typeref:`` field.
 	Enabled by default.
 
-Check the output of the ``--list-fields`` option for the other minor
-fields.
-
 Kinds
-~~~~~~
+......
 
-``kind:`` is a field which represents the *kind* of language object
+``kind`` is a field which represents the *kind* of language object
 specified by a tag. Kinds used and defined are very different between
 parsers. For example, C language defines ``macro``, ``function``,
-``variable``, ``typedef``, etc. See also the descriptions of
-``--list-kinds-full`` option and ``--kinds-<LANG>`` option.
+``variable``, ``typedef``, etc.
+
+``--kinds-<LANG>=[+|-]kinds|*`` option specifies a list of language-specific
+kinds of tags (or kinds) to include in the output file for a particular
+language.
+
+See the output of ``ctags --list-kinds-full`` for the complete
+list of the kinds.
 
-Indicates the type, or kind, of tag. Its value is either one of the
-corresponding one-letter flags described under the various
-``--kinds-<LANG>`` options above, or a long-name flag. It is permitted
+Its value is either one of the
+corresponding one-letter flags or a long-name flag. It is permitted
 (and is, in fact, the default) for the key portion of this field to be
-omitted. The optional behaviors are controlled with the ``--fields`` option.
+omitted. The optional behaviors are controlled with the ``--fields`` option as follows.
+
+.. code-block:: console
+
+	$ ctags -o - kinds.c
```

Comment:
Nice!

Reply:
It will be nice having similar examples for `scope:`.

Reply:
These are just comments. Reflecting on the comment to this pull request is not a must.

Reply:
> It will be nice having similar examples for scope:.

That is a good idea.  I will add it in another pull-request.